technologies/job/trifacta/token-auth/instanceActions.js

```diff
@@ -0,0 +1,274 @@
+          'Authorization': `Bearer ${job.featuresValues.endpoint.access_token}`
+        }
+      }
+    );
+
+    // You can return any payload you want to get in the stop and getStatus functions.
+    return Response.success({ jobGroupId: data.id });
+  } catch (error) {
+    if (error && error.response) {
+      if (
+        error.response.status === 400
+        && error.response.data
+        && error.response.data.exception
+        && error.response.data.exception.name === 'ValidationFailed'
+      ) {
+        return Response.error('Failed to start job from Trifacta : if it\'s the first time that a job is run for the selected wrangled dataset, please complete job execution, profiler and write settings fields', { error: new Error(`${error.response.data.exception.name} - ${error.response.data.exception.message} : ${error.response.data.exception.details}`) });
+      }
+
+      if (error.response.data && error.response.data.exception) {
+        return Response.error(`Failed to start job from Trifacta : ${error.response.data.exception.name} - ${error.response.data.exception.message} : ${error.response.data.exception.details}`, { error: new Error(`${error.response.data.exception.name} - ${error.response.data.exception.message} : ${error.response.data.exception.details}`) });
+      }
+
+      return Response.error(`Failed to start job from Trifacta : ${error.response.status} - ${error.response.statusText}`, { error: new Error(`${error.response.status} - ${error.response.statusText}`) });
+    }
+
+    return Response.error('Failed to obtain job result from Trifacta', { error });
+  }
+};
+
+/**
+ * Logic to retrieve the external job instance status.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data including the payload returned in the start function.
+ */
+exports.getStatus = async ({ job, instance }) => {
+  try {
+    console.log('GET STATUS INSTANCE:', instance);
+
+    const { data } = await axios.get(
+      `${job.featuresValues.endpoint.url}/v4/jobGroups/${instance.payload.jobGroupId}/status`,
+      {
+        httpsAgent: job.featuresValues.endpoint.ignoreSslIssues && job.featuresValues.endpoint.ignoreSslIssues.id ? agent : null,
+        headers: {
+          'Authorization': `Bearer ${job.featuresValues.endpoint.access_token}`
+        }
+      }
+    );
+
+    switch (data) {
+      case 'Created':
+        return Response.success(JobStatus.QUEUED);
+      case 'Pending':
+        return Response.success(JobStatus.QUEUED);
+      case 'InProgress':
+        return Response.success(JobStatus.RUNNING);
+      case 'Complete':
+        return Response.success(JobStatus.SUCCEEDED);
+      case 'Canceled':
+        return Response.success(JobStatus.KILLED);
+      case 'Failed':
+        return Response.success(JobStatus.FAILED);
+      default:
+        return Response.success(JobStatus.AWAITING);
+    }
```

Comment:
Same comment than the swicth in `./technologies/job/trifacta/login-auth/instanceActions.js`